concepticondata/conceptlists.tsv

```diff
@@ -7,6 +7,7 @@ Ray-1895-26	Ray, Sidney H.	1895		26	ultra-stable,areal	English	Britich New Guina
 Backstrom-1992-210a	Backstrom, Peter C. and Radloff, Carla F.	1992	a	210	basic,areal	English	Pakistanian languages		Backstrom1992	Backstrom1992	This list was compiled for a sociolinguistic survey of language use in Pakistan. The list of 210 items is supposed to represent a standardized list used to investigate all languages in the field work survey. However, as can be seen when comparing the official list of 210 items with the concept labels given in the concrete surveys on specific language families, as the list of [Balti dialects](:ref:Backstrom-1992-210b), it becomes obvious that not only some items are missing, but also that certain meanings have been modified, probably in order to make sure that enough words could be elicited. Although all glosses for «verbal meanings» are highly complex, we link them all to the basic concept sets, since from the survey it is clear that the authors adjusted them in order to elicite those verbal forms which represent the root of a given concept most consistently.	209-214	
 Backstrom-1992-210b	Backstrom, Peter C. and Radloff, Carla F.	1992	b	205	basic,areal	English	Balti dialects		Backstrom1992		This is the concrete list which was used for Balti dialects (Tibetan group of Sino-Tibetan) in the large survey of languages in Pakistan (see the [general list of 210 items used in this survey](:ref:Backstrom-1992-210a)). The list is missing five entries and shows also some further differences from the main list. Since we kept the numbering of both the standardized list and the concrete questionnaire used for the Balti dialects, those differences can be easily spotted when comparing the lists.	219-233	
 Bowern-2012-644	Bowern, Claire	2012		644	questionnaire	English	Tasmanian		Bowern2012a		This list supplements a study on Tasmanian languages based on various sources. It provides a linking between different concept labels found in various sources on Tasmanian languages. Due to the specificity of the vocabulary, we had to leave many concepts unlinked.		
+Bowern-2012-348	Bowern, Claire	2012		348	questionnaire	English	Pama-Nyungan	https://zenodo.org/record/1005671	Bowern2012b		This list supplements Bowern & Atkinson (2012) study on Pama-Nyungan languages; it partly overlaps with other concept lists by Bowern.		
```

Comment:
what surprises me greatly is that this list has now 348 items while we were expecting 200, or less, according to the text, from which I linked the Bower-2012-200 (or similar) list originally. Does this mean that the list is a merger of the Swadesh-200 list plus the cultural terms which Bowern addressed in the Hunter Gatherer Project?

Reply:
Also, if writing statements like "overlaps with", it is nice to give a link, such as:

```text
This lists supplements [Bowern and Atkinson's (2012)](:bib:Bowern2012) study and contains material from both Swadesh's [200-item list](:ref:Swadesh-1952-200) and the [questionnaire for hunter gatherer languages](:ref:Bowern-2011-204). 
```
So in a nutshell, we try to figure out how those concepts relate to each other, and I am convinced that this list now contains a merger of Swadesh 200 plus HG vocabulary, as otherwise you could not explain 348 items, although they only talk about less than 200 items in the paper, right? We may decide to just call this list Bowern-2017-348, to indicate that it is a later supplement to the paper, but we might also ask Claire directly, what she thinks.